NCATS_Data_Dictionary.md

```diff
@@ -473,51 +473,51 @@ CyHy stakeholders.
 
 The data in this collection is derived from certificates collected by
 our [Certificate
-Transparency](https: //www.certificate-transparency.org/) log scanner,
+Transparency](https://www.certificate-transparency.org/) log scanner,
 which only grabs certificates that apply to domains in our [domains
-collection](#h.qwt153c8i5hu).  NOTE: More details may be available in
+collection](#h.qwt153c8i5hu).  NOTE:More details may be available in
```

Comment:
Places where we have `NOTE:` should have a space after the colon (there are a few more of these below).

Also, I just noticed that this link to the domains collection is messed up; not sure when it happened, but can you please fix it?  It should look like this:
`[domains collection](domains-collection)`

Reply:
Changed the other items, but confused on domains collection link. I'm showing this (is this right?):
[domains Collection](#domains-collection)